app/routes.py

```diff
@@ -27,22 +30,61 @@ def userRoute(id):
     elif request.method == 'DELETE':
         return user_controller.delete_user(id)
 
+
 @app.route('/login', methods=['POST'])
 def logins():
     return user_controller.login()
 
+
 @app.route('/category', methods=['GET', 'POST'])
 @jwt_required()
 def category():
     if request.method == 'GET':
         return category_controller.get_all_category()
     elif request.method == 'POST':
         return category_controller.create_category()
-    
+
```

Comment:
why this one has 2 line breaks?
could you please format your files using black to make it consistent?

Reply:
done